rust/hyperlane-base/src/contract_sync/mod.rs

```diff
@@ -17,6 +19,8 @@ mod cursor;
 mod eta_calculator;
 mod metrics;
 
+const SLEEP_MS: u64 = 5_000;
```

Comment:
Fwiw Duration::from_millis is const and Copy so we could change line 70's `sleep_duration` to be a `Duration` instead of u64, also that way we don't need this logic `duration.as_millis().try_into().unwrap_or(SLEEP_MS)` on line 99